README.md

```diff
@@ -6,7 +6,7 @@ A service composition is an aggregate of services collectively composed to autom
 
 > This guide walks you through the process of implementing a service composition using Ballerina language. 
 
-The following are the sections available in this guide.
+Following are the sections available in this guide.
```

Comment:
Please revert this change. Using 'The following' in these instances is correct. Please apply elsewhere when you come across this elsewhere.

See https://dictionary.cambridge.org/grammar/british-grammar/following-or-the-following